src/text-distinction/__tests__/text-distinction.js

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import { shallow } from 'enzyme';
+import toJson from 'enzyme-to-json';
+import TextDistinction from '../text-distinction';
+
```

Comment:
we probably need some tests to validate the implementation details.

we can test for the following.
* string splitted correctly ?
*  view renders correctly with front and behind symbol.